### PR TITLE
AA-689: Added ability to display error message when denied access

### PR DESCRIPTION
**Pull request by nkimaina**

### CODE REVIEW

**Comment by nkimaina** [comment]
@mhawila please review

File: app/scripts/authentication/interceptors/authorization.interceptor.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.authentication')
+        .factory('authorizationInterceptor', authorizationInterceptor);
```
Comment (mhawila) [inline app/scripts/authentication/interceptors/authorization.interceptor.js]: @nkimaina stick with the convention of upper case for service names.

File: app/scripts/authentication/authentication.config.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(function () {
+    'use strict';
+
+    var authModule = angular.module('ngAmrsApp');
+    
+    //add authorization interceptor
+    authModule.config(['$httpProvider', function ($httpProvider) {
+        $httpProvider.interceptors.push('authorizationInterceptor');
```
Comment (mhawila) [inline app/scripts/authentication/authentication.config.js]: you don't need a new file for this. You can actually put this one line right after [this line](https://github.com/AMPATH/ng-amrs/blob/master/app/scripts/app.module.js#L396)

**Comment by nkimaina** [comment]
@mhawila corrected the code

**Pull request merged**